FragrantFlowers/STRINGS.cs

```diff
@@ -123,57 +123,57 @@ public class SEEDS
         {
             public class SPINOSA
             {
-                public static LocString SEED_NAME = "Spinosa Black Seed";
-                public static LocString SEED_DESC = "The black seed of a " + UI.FormatAsLink("Spinosa", Plant_SpinosaConfig.ID) + ".";
+                public static LocString SEED_NAME = UI.FormatAsLink("Spinosa Black Seed", Plant_SpinosaConfig.ID.ToUpperInvariant());
+                public static LocString SEED_DESC = $"The black {UI.FormatAsLink("Seed", "PLANTS")} of a " + UI.FormatAsLink("Spinosa", Plant_SpinosaConfig.ID) + ".";
             }
 
             public class DUSKLAVENDER
             {
-                public static LocString SEED_NAME = "Dusk Seed";
-                public static LocString SEED_DESC = "The tiny seed of a " + UI.FormatAsLink("Duskbloom Lavender", Plant_DuskLavenderConfig.ID) + ".";
+                public static LocString SEED_NAME = UI.FormatAsLink("Dusk Seed", Plant_DuskLavenderConfig.ID.ToUpperInvariant());
+                public static LocString SEED_DESC = $"The tiny {UI.FormatAsLink("Seed", "PLANTS")} of a " + UI.FormatAsLink("Duskbloom Lavender", Plant_DuskLavenderConfig.ID) + ".";
             }
 
             public class RIMEDMALLOW
             {
-                public static LocString SEED_NAME = "Iced Mallow Seed";
-                public static LocString SEED_DESC = "The chill seed of a " + UI.FormatAsLink("Iced Mallow", Plant_RimedMallowConfig.ID) + ".";
+                public static LocString SEED_NAME = UI.FormatAsLink("Iced Mallow Seed", Plant_RimedMallowConfig.ID.ToUpperInvariant());
+                public static LocString SEED_DESC = $"The chill {UI.FormatAsLink("Seed", "PLANTS")} of a " + UI.FormatAsLink("Iced Mallow", Plant_RimedMallowConfig.ID) + ".";
             }
         }
 
         public class PLANTS
         {
             public class SPINOSA
             {
-                public static LocString NAME = "Blooming Spinosa";
+                public static LocString NAME = UI.FormatAsLink("Blooming Spinosa", Plant_SpinosaConfig.ID);
                 public static LocString DESC = string.Concat(new string[] { "A rather thorny sten plant that blooms with a beautiful " + UI.FormatAsLink("Spinosa Rose", Crop_SpinosaRoseConfig.ID) + "." });
-                public static LocString DOMESTICATED_DESC = DESC + "/nIn domesticated environment this crop requires copious amounts of" + UI.FormatAsLink("Water", "WATER") + ", and" + UI.FormatAsLink("Dirt", "DIRT") + "as fertilizer. Also requires direct exposure to sunlight.";
+                public static LocString DOMESTICATED_DESC = DESC;
```

Comment:
DOMESTICATED_DESC != DESC, please revert

Reply:
Well, you are the author and have the last word but I dont think this is required. I guess, you did this descriptions as Ronivan did in Dupes Cuisine mod. But this is excessive information. DOMESTICATEDDESC of in-game plants are shorter than DESC to allow you fully see Growth requirement section. But this ones are too big and displace the section. Aaaand it also conflicts with my rebalance branch which I didnt show you yet because I am a little bit afraid of your reaction)).

Reply:
Fixed.